aii-core/src/main/perl/aii-shellfe

```diff
@@ -375,6 +375,8 @@ use constant DHCPPATH  => '/system/aii/dhcp';
 use constant MAC       => '--mac';
 
 
+Readonly my $PROTECTED_COMMANDS   => 'remove|configure|install';
+Readonly my $PROTECTED_OPTION   => 'protectid';
```

Comment:
the commandline option could be renamed to `--confirm`, as the protection feature basically requires you to add a `--yes-i-know-what-i-m-doing confirmationstring`.